unified_planning/model/multi_agent/ma_problem.py

```diff
@@ -150,6 +150,7 @@ def clone(self):
             new_ag._public_fluents = ag._public_fluents.copy()
             new_ag._fluents = ag._fluents.copy()
             new_ag._fluents_defaults = ag._fluents_defaults.copy()
+            new_ag._goals = ag._goals.copy()
```

Comment:
Shouldn't this be _public_goals   and _private_goals?

Reply:
@Alee08 I made one comment about the clone method of ma_problem - could you take a look?

Reply:
Hi @karpase! Thanks for the comment, maybe you saw an "Outdated" commit. In the commit "Minor fix" this is fixed. :)

Reply:
Indeed I missed that, thanks